Documentation/HallSensor.md

```diff
@@ -2,11 +2,11 @@
 
 ## Sleep Mode Menu
 
-In Sleep mode, the iron automatically lowers the temperature to 150 °C (default). This default was chosen as it is just below the melting point of many solders. A stand-by lower temperature helps reduce the rate of oxidation and prevents damage to iron tips. In general, when not using the iron, unplug it or let it sleep to increase the longevity of replaceable tips. The default sleep temperature can be customized.
+In Sleep mode, the iron automatically lowers the temperature to 150 °C (default). This default was chosen as it is just below the melting point of many varities of solder. A stand-by lower temperature helps reduce the rate of oxidation and prevents damage to iron tips. In general, when not using the iron, unplug it or let it sleep to increase the longevity of replaceable tips. The default sleep temperature can be customised to suit how you would like it to act.
 
-Simply moving the iron or pressing any button will wake it back up into soldering mode.
+Simply moving the iron or pressing any button will wake it back up into soldering mode. The sensitivity of this can be adjusted, it is reccomended to adjust this to suit your environment, so that it reliably stays in the sleep mode when not in use but does not go to sleep when in use. (This can vary based on how much movement you are doing when soldering).
```

Comment:
```suggestion
Simply moving the iron or pressing any button will wake it back up into soldering mode. The sensitivity is adjustable. It is recommended to adjust this to suit your environment so that it reliably stays in sleep mode when not in use, but does not go into sleep mode when in use. (This may vary depending on the amount of movement during soldering.)
```